pdc_client/__init__.py

```diff
@@ -97,7 +98,7 @@ class PDCClient(object):
     connections. The authentication token is automatically retrieved (if
     needed).
     """
-    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None):
+    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None, page=None):
```

Comment:
> Having page_size there makes sense since it can be used for all request. But it's only used in get_paged() and not in __getitem__() (I didn't notice this before). This looks like a bug to me.

Yes, currently the page_size is just used by get_paged(), but  the page_size currently can set in plugins instead, for example if you set the page_size in plugin's get list function.
 Like ProductPlugin.product_list, it will get the page_size here.
`    def product_list(self, args):
        filters = {}
        if args.inactive:
            filters['active'] = False
        elif not args.all:
            filters['active'] = True
        filters['page_size'] = True

        products = self.client.get_paged(self.client["products"]._, **filters)`

So I said the page_size here is same like page.

And with another patch I committed, it should be:
`
client = pdc_client.PDCClient("upstream", is_multi=True)

a = client["releases"](page_size=30)
`